int32 newRightSpeed
int32 newLeftSpeed
---
int32 newRightSpeed
int32 newLeftSpeed
